src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -276,9 +285,37 @@ export default {
         }
 
         // Render the details slot in a TD
-        const $details = h(BTd, { props: { colspan: fields.length }, class: this.detailsTdClass }, [
-          this.normalizeSlot(detailsSlotName, detailsScope)
-        ])
+        var $details = !detailsScope.fields.some(e => e.details)
```

Comment:
use `this.computedFields.some(...)`

Reply:
`class: this.detailsTdClass` is also missing.
Thats why the tests are failing.

Reply:
Got it, thks